flink-table/flink-table-planner/pom.xml

```diff
@@ -347,6 +367,7 @@ under the License.
 							<include>com.google.guava:failureaccess</include>
 							<include>commons-codec:commons-codec</include>
 							<include>commons-io:commons-io</include>
+							<include>org.checkerframework:checker-qual</include>
```

Comment:
I fear it will not work after relocation because of this line at `core/src/main/java/org/apache/calcite/util/ImmutableBeans.java`
```java
final boolean required = propertyType.isPrimitive()
          || !hasAnnotation(
              method.getAnnotatedReturnType(),
          "org.checkerframework.checker.nullness.qual.Nullable");
```
i will also double check it

Reply:
you're right add relocation as well